docs/docs/docs/getting-started/installation.md

```
@@ -197,31 +197,20 @@ Click on the image below to play the video.
    ```
-1. Create the `.env` file by copying the template from the `envFiles/` directory
-   ```bash
-   cp envFiles/.env.devcontainer .env
-   ```
-1. Change the `.env` file's `API_POSTGRES_HOST` parameter to `localhost`.
-   ```ini
-   API_POSTGRES_HOST=localhost
-   ```
+1. Create the `.env` file by copying the template from the `envFiles/` directory.
+   1. **DO NOT EDIT EITHER FILE!**
+      ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)